Tests/PenguinStructuresTests/ProjectionTests.swift

```diff
@@ -0,0 +1,89 @@
+//******************************************************************************
+// Copyright 2020 Penguin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import XCTest
+import PenguinStructures
+
+struct Pair<T,U> {
```

Comment:
Any reason not to use Tuple2?

Reply:
I didn't think of it.  Hindsight justification: this is simpler, makes the test more self-contained, and allows us to extend Pair for testing without interacting with any other tests.  That said, please LMK if you want me to change it.